serverless/admin_guide/serverless-configuration.adoc

```diff
@@ -0,0 +1,58 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-configuration"]
+= Configuring {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: serverless-configuration
+
+toc::[]
+
+The {ServerlessOperatorName} manages the *global* configuration of a Knative installation, including propagating values from the `KnativeServing` and `KnativeEventing` custom resources to system link:https://kubernetes.io/docs/concepts/configuration/configmap/[config maps].
```

Comment:
Usually asterisks are used for GUI items like menus, buttons, page titles, etc., as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting. I'm not entirely sure, but is this one of those?

Reply:
I think I actually did this by mistake :) I'll fix it to be italics, which would be correct for a first instance of a term, is that right?